index.html

```diff
@@ -49,8 +49,187 @@ <h1 class="username">I'm Mark, Glad to see you!</h1>
+                <div class="name">
+                    <h1 class="username">About Myself</h1>
+                </div>
+                <div class="description">
+                    <p class="user-description">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                <span class="socialmedia">LET'S CONNECT</span>
+                <div class="social-media-icons">
+                    <a href="https://twitter.com/Eng_Ngwala" class="twitter-link">
+                        <img src="./images/twitter.svg" alt="twitter" class="twitter" width="20" height="20">
+                    </a>
+                    <a href="https://www.linkedin.com/in/mark-otweyo-034752123/" class="linkedin-link">
+                        <img src="./images/linkedin.svg" alt="linkedin" class="linkedin" width="20" height="20">
+                    </a>
+                    <a href="mailto:markotweyo@gmail.com" class="email-link">
+                        <img src="./images/email.svg" alt="email" class="email" width="20" height="20">
+                    </a>
+                    <a href="https://github.com/Markotweyo" class="social-media-link">
+                        <img src="./images/github.svg" alt="github" class="github" width="20" height="20">
+                    </a>
+                </div>
+            </div>
+            <div class="button">
+                <button class="get-resume-button" type="button">Get my resume</button>
+            </div>
+            <div class="skill-list">
+                <ul class="skills-list_frame">
+                    <li>
+                        <div class="language-container">
+                            <a class="languages">Languages</a>
+                            <img src="./images/downarrow.svg" alt="" class="arrow" width="40" height="40">
+                        </div>
+                        <ul class="language_items">
+                            <li class="javascript">
+                                <img src="./images/js.svg" alt="" class="" width="48" height="48">
+                                <a href="#">Javascript</a>
+                            </li>
+                            <li class="html">
+                                <img src="./images/html.svg" alt="" class="" width="48" height="48">
+                                <a href="#">HTML</a>
+                            </li>
+                            <li class="css">
+                                <img src="./images/css.svg" alt="" class="" width="48" height="48">
+                                <a href="#">CSS</a>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <div class="framework-container">
```

Comment:
- [ ] Kindly make sure that your design matches that of the Figma design as much as possible. the width of the container here does not match that of the Figma design. Please fix it.
Your design

![Capture](https://user-images.githubusercontent.com/85095702/170707489-851e2a13-7aa9-413b-969f-0e5d83a9c775.jpg)

Figma Design
![figma](https://user-images.githubusercontent.com/85095702/170707591-0247edd1-49ff-464a-9c9b-310d6933c970.jpg)